app/src/main/AndroidManifest.xml

```diff
@@ -10,20 +10,18 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@mipmap/ic_broccoli"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@mipmap/ic_broccoli_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Oipie"
-        tools:targetApi="31">
+        android:theme="@style/Theme.Oipie.Starting"
+        tools:targetApi="33">
```

Comment:
Is this upgrade of target really needed for the splash?

Reply:
Nope, but, why don't upgrade to the latest API? I mean, these changes are exclusive for android 12 and 13, I don't see any purpose to maintain higher API version but not the highest stable if is just 2 versions above.

But whatever, is just a dumb change.